yarn.lock

```diff
@@ -20155,11 +20150,6 @@ minimist@^0.1.0:
   resolved "https://registry.yarnpkg.com/minimist/-/minimist-0.1.0.tgz#99df657a52574c21c9057497df742790b2b4c0de"
   integrity sha1-md9lelJXTCHJBXSX33QnkLK0wN4=
 
-minimist@^1.2.5:
-  version "1.2.5"
-  resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.5.tgz#67d66014b66a6a8aaa0c083c5fd58df4e4e97602"
-  integrity sha512-FM9nNUYrRBAELZQT3xeZQ7fmMOBg6nWNmJKTcgsJeaLstP/UODVpGsr5OhXhhXg6f+qtJ8uiZ+PUxkDWcgIXLw==
-
 minimist@~0.0.1:
   version "0.0.10"
```

Comment:
Looks like there's still a transitive dependency that's resolving to `0.0.10`?

Reply:
This is partly because of `@elastic/makelogs@5.0.0`. I've later published v5.0.1 of this module have upgraded that in `master` in #61396. Once I rebase part of it should be taken care of.

The other retainers are `handlebars@4.5.3` and `karma@3.1.4`. I'm in the process of upgrading `handlebars`, but `karma` doesn't have an update available that solves this, so I'll add that to the `resolutions` list.

Reply:
Correction, the new version of handlebars that fixes this hasn't been released yet. So we'll have to force-upgrade handlebars version of mimimist as well

Reply:
Fixed by force-upgrading `optimist`